examples/simple/main.tf

```diff
@@ -27,7 +27,7 @@ module "secret-manager" {
     {
       name                  = "secret-1"
       secret_data           = "secret information"
-      automatic_replication = true
```

Comment:
@apeabody The argument deprecated in [v4.83.0](https://github.com/hashicorp/terraform-provider-google/releases/tag/v4.83.0). So it will break versions of this version and below.

Reply:
not sure, let me test that and come back with an answer

Reply:
@apeabody Deployed a secret with the "old" way and after that upgraded the module to the one in the PR and changed the attribute as in this example and got:
`No changes. Your infrastructure matches the configuration.` 
So, it shouldn't break existing resources.

Reply:
Great - Thanks!